<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditPAT</class>
 <widget class="QDialog" name="DialogEditPAT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PAT Edit</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBoxDepthSortedBin">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineEditName</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QRadioButton" name="radioButtonXYZ">
     <property name="text">
      <string>XYZ</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupConvention</string>
     </attribute>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Position</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>X [m]:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>spinBoxPositionX</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="spinBoxPositionX">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-9999999.998999999836087</double>
          </property>
          <property name="maximum">
           <double>9999999.998999999836087</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Y [m]:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>spinBoxPositionY</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spinBoxPositionY">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-9999999.998999999836087</double>
          </property>
          <property name="maximum">
           <double>9999999.998999999836087</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Z [m]:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>spinBoxPositionZ</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="spinBoxPositionZ">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-9999999.998999999836087</double>
          </property>
          <property name="maximum">
           <double>9999999.998999999836087</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Attitude</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>X [deg]:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>spinBoxAttitudeX</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="spinBoxAttitudeX">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Y [deg]:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>spinBoxAttitudeY</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spinBoxAttitudeY">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Z [deg]:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>spinBoxAttitudeZ</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="spinBoxAttitudeZ">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>-360.000000000000000</double>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>379</width>
       <height>205</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Convention:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QCheckBox" name="checkBoxDepthSortedBin">
     <property name="text">
      <string>Depth Sorted Bin</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Depth Sorted Bin:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QRadioButton" name="radioButtonZYX">
     <property name="text">
      <string>ZYX</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroupConvention</string>
     </attribute>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QPushButton" name="pushButtonFromGeo">
     <property name="text">
      <string>From Geographic Coordinates</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>spinBoxPositionX</tabstop>
  <tabstop>spinBoxPositionY</tabstop>
  <tabstop>spinBoxPositionZ</tabstop>
  <tabstop>spinBoxAttitudeX</tabstop>
  <tabstop>spinBoxAttitudeY</tabstop>
  <tabstop>spinBoxAttitudeZ</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogEditPAT</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogEditPAT</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroupConvention"/>
 </buttongroups>
</ui>
